app/views/shared/_footer_lite.html.slim

```diff
@@ -3,9 +3,9 @@ footer.footer.bg-navy.white
     .clearfix
       .sm-col = t('shared.footer_lite.gsa')
       .sm-col-right.caps
-        = link_to t('links.help'), help_path,
+        = link_to t('links.help'), "https://#{APP_NAME}/help",
```

Comment:
We're only going to have one marketing site. Should I just hardcode `login.gov` here instead?